src/Adornment/styled.ts

```diff
@@ -0,0 +1,31 @@
+import styled, { css } from 'styled-components'
```

Comment:
I'd opt to simply include these statements in the main Adornment.tsx.
wdyt @Michiel87 ?